tests/test_views_search.py

```diff
@@ -226,3 +230,16 @@ def test_query_wrong(app, indexed_records, search_url):
 
         res = client.get(search_url, query_string={'q': 'test;'})
         assert res.status_code == 200
+
+
+def test_elasticsearch_exception(app, indexed_records, search_url):
+    with app.test_client() as client:
+        for error_type in elasticsearch_error_handlers.keys():
+            es_req_error = RequestError(
+                '', '', {'error': {'root_cause': [{'type': error_type}]}})
+            with patch('invenio_records_rest.views.RecordsListResource.get',
```

Comment:
Can you make a request that really throws the exception instead of just patching the get method? Otherwise we won't be able to detect that the exception name has changed.